parse.py

```diff
@@ -217,19 +217,15 @@ def evaluate_stack(self: object, symbol_stack: list) -> str:
         if operation:
             return operation()
         # If symbol is not an operation or a constant, the symbol must be a numeral. Try casting to an int.
```

Comment:
You just need to put this comment back above the try statement below. That's the code that casts to an int. You could split the comment in two so that the 'If symbol is not an operation or a constant, the symbol must be a numeral.' is left here and the 'Try casting to an int.' is placed below.

Reply:
Just updated it